meinberlin/apps/contrib/templates/meinberlin_contrib/components/alert.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}
+
+<div class="alert alert--{{ alert_type }}" role="alert" aria-live="{% if alert_type == 'info' or alert_type == 'success' %}polite{% elif alert_type == 'danger' %}assertive{% endif %}" aria-atomic="true">
+  <div class="alert__content">
+    {% if alert_headline %}
```

Comment:
we could also add `aria-describedby` pointing to the headline or text for better screen reader guidance
```
{% if alert_headline %}aria-describedby="alert-headline"{% endif %}>
  <div class="alert__content">
    {% if alert_headline %}
      <h3 id="alert-headline" class="alert__headline">{% translate alert_headline %}</h3>
    {% endif %}
```

Reply:
not sure if I understand... the screen reader reads the headline and text anyway, right?

Reply:
Screen readers will read the content within the alert, but adding aria-describedby explicitly links the alert to its headline or text. This improves accessibility by clarifying that the headline is part of the alert, providing better context and guidance for users who rely on screen readers.

Reply:
okay thanks, understood. I will change acordingly